CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Due to [\#5553](https://github.com/decidim/decidim/pull/5553), SSL is turned on
 ### Added
 
 - **decidim-core**: Allow groups to have private conversations with other users or groups. [\#6009](https://github.com/decidim/decidim/pull/6009)
+- **decidim-api**: Use organization time zone [\#6088](https://github.com/decidim/decidim/pull/6088)
```

Comment:
Could you check this log entry please? 
Although I think this changelog entry should already be in develop branch, it looks as it doesn't and you're uploading it just now... :confused:

Reply:
Are you sure? I dont' see this change.